templates/scenarios/ts/link-unfurling/appPackage/manifest.json.tpl

```diff
@@ -0,0 +1,70 @@
+{
+    "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.16/MicrosoftTeams.schema.json",
+    "manifestVersion": "1.16",
+    "version": "1.0.0",
+    "id": "${{TEAMS_APP_ID}}",
+    "packageName": "com.microsoft.teams.extension",
+    "developer": {
+        "name": "Teams App, Inc.",
+        "websiteUrl": "https://www.example.com",
+        "privacyUrl": "https://www.example.com/termofuse",
+        "termsOfUseUrl": "https://www.example.com/privacy"
+    },
+    "icons": {
+        "color": "color.png",
+        "outline": "outline.png"
+    },
+    "name": {
+        "short": "{{appName}}-${{TEAMSFX_ENV}}",
+        "full": "full name for {{appName}}"
+    },
+    "description": {
+        "short": "short description of {{appName}}",
+        "full": "full description of {{appName}}"
+    },
+    "accentColor": "#FFFFFF",
+    "bots": [],
+    "composeExtensions": [
+        {
+            "botId": "${{BOT_ID}}",
+            "commands": [
```

Comment:
The `composeExtensions` must have at least 1 commands. We can discuss this offline.

Reply:
Got it.

Reply:
That's a little weird at the first glance. Is there any Teams sample can refer to?

Reply:
Although I find this in Teams Sample https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/main/samples/msgext-link-unfurling/nodejs/bots/teamsLinkUnfurlingBot.js. It include both link unfurling and search in link unfurling sample.

Can we check with Junjie to see if we need to merge link unfurling with search query in one scenario, or ask the recommendation from ME team?